src/blister.h

```diff
@@ -140,6 +140,9 @@ static inline UInt * BLOCKS_BLIST(Obj list)
     return BLOCKS_BLIST_UNSAFE(list);
 }
 
+static inline const UInt * CONST_BLOCKS_BLIST(Obj list) {
+  return (const UInt *)BLOCKS_BLIST(list);
```

Comment:
I'm afraid this implementation defeats the ultimate purpose: to allow inserting read/write guards in an "optimal" fashion. Really need this (and use 4 spaces for indent):
```C
static inline const UInt * CONST_BLOCKS_BLIST(Obj list)
{
    GAP_ASSERT(IS_BLIST_REP_WITH_COPYING(list));
    return ((const UInt *)(CONST_ADDR_OBJ(list) + 1));
}
```